simpleflow/execute.py

```diff
@@ -158,14 +159,24 @@ def execute(*args, **kwargs):
                     exception = pickle.loads(
                         base64.b64decode(excline.rstrip()))
                 except TypeError:
-                    cls, msg = exclines[-1].split(':', 1)
-                    exception = eval('{}("{}")'.format(
-                        cls.strip(),
-                        msg.strip(),
-                    ))
+                    excline = exclines[-1]
+                    exception = Exception(excline)
+                    if ':' in excline:
+                        cls, msg = excline.split(':', 1)
+                        if re.match(r'\s*[\w.]+\s*', cls):
+                            try:
+                                exception = eval('{}("{}")'.format(
+                                    cls.strip(),
+                                    msg.strip(),
+                                ))
+                            except:
+                                pass
 
                 raise exception
-            return json.loads(output)
+            try:
+                return json.loads(output)
+            except:
+                pass
```

Comment:
Yep; we'd better print the failure, hopping it'll end in papertrail (so long as we don't fill it again...)